app/javascript/redux/messageReducer.js

```diff
@@ -0,0 +1,30 @@
+const initialState = {
+  id: 13,
+  greet: 'Hey, how are you doing?',
+  created_at: '2022-09-20T21:27:23.675Z',
+  updated_at: '2022-09-20T21:27:23.675Z',
+};
+
+const messageReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case 'ISLOADED':
+      return action.payload;
+    default:
+      return state;
+  }
+};
+
+// action
+
+export const onFectchOk = (data) => ({
+  type: 'ISLOADED',
+  payload: data,
+});
+
+// Thunk function to handle api calls
+export const fetchGreet = () => async (dispatch) => {
+  const response = await fetch('http://localhost:3000/api/v1/messages');
```

Comment:
Rhere is a store, an action , and a reducer that uses the url to the endpoint from the Rails API 😉📝 Awesome!

Reply:
Great cheers 👍🏽